assets/javascripts/discourse/initializers/extend-for-assigns.js

```diff
@@ -680,6 +700,7 @@ function initialize(api) {
   api.decorateWidget("post-contents:after-cooked", (dec) => {
     const postModel = dec.getModel();
     if (postModel) {
+      // 'name' is not empty in the attrs ^, looking good
```

Comment:
ope this one's my bad for leaving it in 😅

Reply:
lol I didnt catched it